scaleway/resource_rdb_instance_test.go

```diff
@@ -49,9 +49,9 @@ func TestAccScalewayRdbInstance_Basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: `
-					resource scaleway_rdb_instance main {
-						name = "test-rdb"
-						node_type = "db-dev-s"
+					resource scaleway_rdb_instance rdbBasic {
```

Comment:
Why is this rename needed?

Reply:
If you don't have several resources, just naming them `main` or something generic is enough.